### PR TITLE
Add 3rd party tool - qStudio.

### PR DESCRIPTION
**Pull request by ryanhamilton**
A free SQL querying tool that allows time-series charting.

### CODE REVIEW

File: docs/third-party-tools/qstudio.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+2. Launch qStudio.
+3. Go to `Server->Add Server`.
+4. Click `Add data source`
+5. Choose the `PostgreSQL` plugin and configure it with the following settings:
+
+```
+host:localhost
+port:8812
+database:qdb
+user:user
```
Comment (javier) [inline docs/third-party-tools/qstudio.md:27]: user needs to be `admin`

Comment (quankaroo) [reply docs/third-party-tools/qstudio.md]: Good catch. Fixed.

**Review by javier — APPROVED** [review]
Thanks!

File: docs/third-party-tools/qstudio.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+database:qdb
+user:admin
+password:quest
+```
+
+## Sending Queries
+
+Queries can be ran by either pressing Ctrl+Enter to run the current line or by pressing Ctrl+E to run the highlighted code.
+
+![Screenshot of the qStudio UI query](/img/docs/guide/qstudio/qstudio-query.png)
```
Comment (argshook) [inline docs/third-party-tools/qstudio.md:35]:
```suggestion
![Screenshot of the qStudio UI query](/img/guides/qstudio/qstudio-query.png)
```

**Review by argshook — CHANGES_REQUESTED** [review]
hey @ryanhamilton thanks a lot for contributing!

there's one issue with screenshot image path, i have pushed a code suggestion. Once fixed, we can merge this PR.

Thank you!

**Comment by ryanhamilton** [comment]
Merged your suggestion, thanks Arijus. 
I also just learnt about github suggestions today, very cool.

**Review by argshook — APPROVED** [review]

**Comment by argshook** [comment]
> Merged your suggestion, thanks Arijus.

awesome, thanks!

> I also just learnt about github suggestions today, very cool.

If you don't know about the dot yet, it should be impress you!

try pressing `.` on your keyboard when browsing some github page (like this one)

**Pull request merged**